packages/styles/scss/components/card-link/_card-link.scss

```diff
@@ -32,9 +33,13 @@
       }
     }
   }
+  ::slotted([slot='heading']),
   .#{$prefix}--card__CTA.#{$prefix}--card .#{$prefix}--card__copy {
```

Comment:
Looking at this design specs doc, it looks like we're using `$body-long-02` for the copy, unless this doc is out of date. cc: @shixiedesign 

https://ibm.ent.box.com/file/827717793884